### PR TITLE
CORE-411 - Teamspeak Controller Idle Exemption

### PR DESCRIPTION
**Pull request by atoff**
Closes CORE-411

### CODE REVIEW

File: app/Libraries/TeamSpeak.php [hunk 1]
```diff
@@ -376,7 +376,7 @@ public static function checkClientIdleTime(TeamSpeak3_Node_Client $client, Accou
     {
         $idleTime = floor($client['client_idle_time'] / 1000 / 60); // minutes
 
-        if ($member->hasPermissionTo('teamspeak/idle/permanent')) {
```
Comment (AxonC) [inline app/Libraries/TeamSpeak.php]: Can we go further with this check? At present, they could not be on a UK position.

Comment (atoff) [reply app/Libraries/TeamSpeak.php]: I can't find anything to suggest this? It should just check if they have an active ATC session regardless of where it is?

Comment (AxonC) [reply app/Libraries/TeamSpeak.php]: I was just taking that from the JIRA story

Comment (atoff) [reply app/Libraries/TeamSpeak.php]: Ah wording subtleties... I will amend to user ;)

**Pull request merged**